data/moves.js

```diff
@@ -11142,7 +11142,7 @@ exports.BattleMovedex = {
 		name: "Rain Dance",
 		pp: 5,
 		priority: 0,
-		flags: {},
+		flags: {dance: 1},
```

Comment:
Hmm, you're right. I'll check it after I'm done dling (or Marty can comment on this)
Also now i do realise, I also need to check JAP names